.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+#### Ticket (For Vimeo staff only)
+
+[TICKET_NUMBER](https://vimean.atlassian.net/browse/TICKET_NUMBER)
+
+#### Issue Summary
+
+- *Outline the goals of the PR in technical terms. If this branch fixes a bug, state what caused the bug.*
+
+#### Pull Request Checklist
+
+- [ ] Resolved any merge conflicts
+- [ ] No build errors or warnings are introduced
+- [ ] New files are written in Swift
+- [ ] New classes contain license headers
+- [ ] New classes have Documentation
+- [ ] New public methods have Documentation
+
+#### Implementation Summary
+
+- *Identify the core components that were added or changed, and briefly state the reasoning behind these changes.*
+
+- *List any key assumptions made.*
+
+#### Reviewer Tips
+
+- *Provide the order in which files or components should be reviewed.* 
+
+- *List general components of the application that this PR will affect.*
```

Comment:
Not sure the term `application` applies in this sense, since this is a library itself. Perhaps `List general components that this PR will affect.` would be better suited.